From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 01:26:38 +0100
Subject: Avoid GCC warning suggesting parentheses

---
 xsec/utils/XSECDOMUtils.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/xsec/utils/XSECDOMUtils.cpp b/xsec/utils/XSECDOMUtils.cpp
index 4937434..486a33d 100644
--- a/xsec/utils/XSECDOMUtils.cpp
+++ b/xsec/utils/XSECDOMUtils.cpp
@@ -709,9 +709,9 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 
 static bool isHexDigit(const XMLCh toCheck)
 {
-    if ((toCheck >= chDigit_0) && (toCheck <= chDigit_9)
-    ||  (toCheck >= chLatin_A) && (toCheck <= chLatin_F)
-    ||  (toCheck >= chLatin_a) && (toCheck <= chLatin_f))
+    if ((toCheck >= chDigit_0 && toCheck <= chDigit_9)
+    ||  (toCheck >= chLatin_A && toCheck <= chLatin_F)
+    ||  (toCheck >= chLatin_a && toCheck <= chLatin_f))
     {
         return true;
     }
