From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 01:19:01 +0100
Subject: Fix indentation 5

---
 xsec/xkms/impl/XKMSRecoverResultImpl.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xsec/xkms/impl/XKMSRecoverResultImpl.cpp b/xsec/xkms/impl/XKMSRecoverResultImpl.cpp
index 4cdbfba..1f90870 100644
--- a/xsec/xkms/impl/XKMSRecoverResultImpl.cpp
+++ b/xsec/xkms/impl/XKMSRecoverResultImpl.cpp
@@ -324,8 +324,8 @@ XENCEncryptedData * XKMSRecoverResultImpl::setRSAKeyPair(const char * passPhrase
 		}
 		uri = algorithmSB.sbStrToXMLCh();
 	}
-    else
-        uri = algorithmURI;
+	else
+		uri = algorithmURI;
 
 	// Find if we can get an algorithm for this URI
 	XSECAlgorithmHandler *handler;
