From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 00:40:44 +0100
Subject: Fix indentation 2

---
 xsec/xkms/impl/XKMSRegisterResultImpl.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/xsec/xkms/impl/XKMSRegisterResultImpl.cpp b/xsec/xkms/impl/XKMSRegisterResultImpl.cpp
index d51f2ef..36bc0ee 100644
--- a/xsec/xkms/impl/XKMSRegisterResultImpl.cpp
+++ b/xsec/xkms/impl/XKMSRegisterResultImpl.cpp
@@ -324,8 +324,8 @@ XENCEncryptedData * XKMSRegisterResultImpl::setRSAKeyPair(const char * passPhras
 		}
 		uri = algorithmSB.sbStrToXMLCh();
 	}
-    else
-        uri = algorithmURI;
+	else
+		uri = algorithmURI;
 
 	// Find if we can get an algorithm for this URI
 	XSECAlgorithmHandler *handler;
@@ -342,10 +342,10 @@ XENCEncryptedData * XKMSRegisterResultImpl::setRSAKeyPair(const char * passPhras
 	unsigned char kbuf[XSEC_MAX_HASH_SIZE];
 	unsigned int len = CalculateXKMSKEK((unsigned char *) passPhrase, (int) strlen(passPhrase), kbuf, XSEC_MAX_HASH_SIZE);
 
-    if (len == 0) {
+	if (len == 0) {
 		throw XSECException(XSECException::XKMSError,
 			"XKMSRegisterResult::setRSAKeyPair - error deriving KEK");
-    }
+	}
 
 
 	XSECCryptoKey * sk = handler->createKeyForURI(
