From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 00:46:26 +0100
Subject: Fix indentation 3

---
 xsec/enc/OpenSSL/OpenSSLCryptoKeyEC.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xsec/enc/OpenSSL/OpenSSLCryptoKeyEC.cpp b/xsec/enc/OpenSSL/OpenSSLCryptoKeyEC.cpp
index 189bc0f..3233343 100644
--- a/xsec/enc/OpenSSL/OpenSSLCryptoKeyEC.cpp
+++ b/xsec/enc/OpenSSL/OpenSSLCryptoKeyEC.cpp
@@ -307,8 +307,8 @@ XSECCryptoKey * OpenSSLCryptoKeyEC::clone() const {
 	XSECnew(ret, OpenSSLCryptoKeyEC);
 
 	ret->m_keyType = m_keyType;
-    if (mp_ecKey)
-        ret->mp_ecKey = EC_KEY_dup(mp_ecKey);
+	if (mp_ecKey)
+		ret->mp_ecKey = EC_KEY_dup(mp_ecKey);
 
 	return ret;
 
