From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sat, 30 Jan 2016 21:55:41 +0100
Subject: Fix GCC6 FTBFS

---
 xsec/tools/checksig/InteropResolver.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xsec/tools/checksig/InteropResolver.cpp b/xsec/tools/checksig/InteropResolver.cpp
index e700b99..f35e087 100644
--- a/xsec/tools/checksig/InteropResolver.cpp
+++ b/xsec/tools/checksig/InteropResolver.cpp
@@ -645,7 +645,7 @@ XSECCryptoKey * InteropResolver::resolveKey(DSIGKeyInfoList * lst) {
 
 	}
 
-	return false;
+	return NULL;
 
 }
 
