From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 00:59:52 +0100
Subject: Fix indentation 4

---
 xsec/tools/c14n/c14n.cpp | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/xsec/tools/c14n/c14n.cpp b/xsec/tools/c14n/c14n.cpp
index c3415e9..16315cc 100644
--- a/xsec/tools/c14n/c14n.cpp
+++ b/xsec/tools/c14n/c14n.cpp
@@ -179,17 +179,17 @@ int main(int argc, char **argv) {
     }
 
 	// Create the canonicalizer
-    XSECC14n20010315* canon=NULL;
-    if (subtree)
-	    canon = new XSECC14n20010315(theDOM, subtree);
-    else
-        canon = new XSECC14n20010315(theDOM);
+	XSECC14n20010315* canon=NULL;
+	if (subtree)
+		canon = new XSECC14n20010315(theDOM, subtree);
+	else
+		canon = new XSECC14n20010315(theDOM);
 	canon->setCommentsProcessing(printComments);
 	canon->setUseNamespaceStack(true);
-    if (inclusive11)
-        canon->setInclusive11();
-    else if (exclusive)
-        canon->setExclusive();
+	if (inclusive11)
+		canon->setInclusive11();
+	else if (exclusive)
+		canon->setExclusive();
 
 	// canon->XPathSelectNodes("(/descendant-or-self::node() | /descendant-or-self::node()/attribute::* | /descendant-or-self::node()/namespace::*)[ self::ietf:e1 or (parent::ietf:e1 and not(self::text() or self::e2)) or count (id(\"E3\") | ancestor-or-self::node()) = count (ancestor-or-self::node())]");
 
