From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Thu, 21 Jul 2016 19:24:02 +0200
Subject: Only add found packages to the pkg-config dependencies

---
 m4/ax_pkg_check_modules.m4 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/m4/ax_pkg_check_modules.m4 b/m4/ax_pkg_check_modules.m4
index f3af0f6..ab8adf9 100644
--- a/m4/ax_pkg_check_modules.m4
+++ b/m4/ax_pkg_check_modules.m4
@@ -58,10 +58,10 @@ AC_DEFUN([AX_PKG_CHECK_MODULES],[
     m4_define([ax_package_requires_private],
               [m4_default_quoted([$7],[AX_PACKAGE_REQUIRES_PRIVATE])])
 
-    ax_package_requires="$[]ax_package_requires $2"
-    ax_package_requires_private="$[]ax_package_requires_private $3"
-
-    PKG_CHECK_MODULES([$1],[$2 $3],[$4],[$5])
+    PKG_CHECK_MODULES([$1],[$2 $3],[
+        ax_package_requires="$[]ax_package_requires $2"
+        ax_package_requires_private="$[]ax_package_requires_private $3"
+        $4],[$5])
 
     # Substitute output.
     AC_SUBST(ax_package_requires)
