From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 31 Jan 2016 00:30:22 +0100
Subject: Remove unused dereferences

---
 xsec/utils/XSECDOMUtils.cpp | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/xsec/utils/XSECDOMUtils.cpp b/xsec/utils/XSECDOMUtils.cpp
index 848923d..4937434 100644
--- a/xsec/utils/XSECDOMUtils.cpp
+++ b/xsec/utils/XSECDOMUtils.cpp
@@ -492,14 +492,14 @@ XMLCh * encodeDName(const XMLCh * toEncode) {
 	// Find where the trailing whitespace starts
 	const XMLCh * ws = &toEncode[XMLString::stringLen(toEncode)];
 
-	*ws--;
+	ws--;
 	while (ws != toEncode &&
 		(*ws == '\t' || *ws == '\r' || *ws ==' ' || *ws == '\n'))
-		*ws--;
+		ws--;
 
 	// Set to first white space character, if we didn't get back to the start
 	if (toEncode != ws)
-		*ws++;
+		ws++;
 
 	// Now run through each character and encode if necessary
 
@@ -537,9 +537,9 @@ XMLCh * encodeDName(const XMLCh * toEncode) {
 
 			// Determine if this is an RDN separator
 			const XMLCh *j = i;
-			*j++;
+			j++;
 			while (*j != chComma && *j != chEqual && *j != chNull)
-				*j++;
+				j++;
 
 			if (*j != chEqual)
 				result.sbXMLChAppendCh(chBackSlash);
@@ -564,7 +564,7 @@ XMLCh * encodeDName(const XMLCh * toEncode) {
 
 		}
 
-		*i++;
+		i++;
 
 	}
 
@@ -576,7 +576,7 @@ XMLCh * encodeDName(const XMLCh * toEncode) {
 		else
 			result.sbXMLChAppendCh(*i);
 
-		*i++;
+		i++;
 
 	}
 
@@ -603,8 +603,8 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 	if (*i == chBackSlash && i[1] == chPound) {
 
 		result.sbXMLChAppendCh(chPound);
-		*i++;
-		*i++;
+		i++;
+		i++;
 
 	}
 
@@ -612,11 +612,11 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 
 		if (*i == chBackSlash) {
 
-			*i++;
+			i++;
 
 			if (*i == chDigit_0) {
 
-				*i++;
+				i++;
 
 				if (*i >= chDigit_0 && *i <= chDigit_9) {
 					result.sbXMLChAppendCh(*i - chDigit_0);
@@ -635,7 +635,7 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 
 			else if (*i == chDigit_1) {
 
-				*i++;
+				i++;
 
 				if (*i >= chDigit_0 && *i <= chDigit_9) {
 					result.sbXMLChAppendCh(16 + *i - chDigit_0);
@@ -654,7 +654,7 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 
 			else if (*i == chDigit_2) {
 
-				*i++;
+				i++;
 
 				if (*i == '0') {
 					result.sbXMLChAppendCh(' ');
@@ -685,7 +685,7 @@ XMLCh * decodeDName(const XMLCh * toDecode) {
 
 			}
 
-			*i++;
+			i++;
 
 		}
 
